NOTICE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2022 Elasticsearch BV
+Copyright 2023 Elasticsearch BV
```

Comment:
Shouldn't it be `2022-2023`?

```suggestion
Copyright 2022-2023 Elasticsearch BV
```

Reply:
Nope, the template says `currentYear`: https://github.com/elastic/apm-aws-lambda/blob/main/scripts/templates/NOTICE.txt.tmpl#L13

Reply:
I'm not sure what we're doing in other repositories. It might be worth to look into it and make it more consistent

Reply:
The APM Server used that format, that's why I was suggesting we use that format here as well.

Reply:
I've looked at a few repositories and I think the lambda extension is the only one using `currentYear`. I'll gather more data and open a followup issue/PR.